13-collabs.Rmd

```diff
@@ -5,6 +5,248 @@
+
+Remember that you are not your code and mistakes are all a part of the process! Putting your project out there can feel a tad vulnerable even. You may have felt the impulse to keep your code's problems buried under a rug, but you pushed past that and are making your analyses transparent! Remember that hidden problems don't get solved, but known problems are opportunities for reaching an even better end result than you imagined!
+
+```{r, fig.alt="Remember that you are not your code and mistakes are all a part of the process! The cartoon shows you with exclamation point and equal sign with a fake bit of code to illustrate you are not your code.", out.width = "100%", echo = FALSE}
+ottrpal::include_slide("https://docs.google.com/presentation/d/1MNHf8JpolaEP_vQ_kB-1xRBF9wo3haCArRu117hBoHA/edit#slide=id.g33bf0789107_300_2625")
+```
+
+When you receive a review back remember that you and the reviewer are on the same team and both want the best end result feasible for this project! They may suggest ideas that you love and can't wait to implement. They also might suggest ideas you don't agree with. Do your best to take all their comments as positive learning opportunities and look for ways to compromise and determine solutions collaboratively.
+
+## Engaging in Code Review - as a reviewer
+
+When reviewing a pull request, you take on responsibility to ensure that the pull request is getting the project to a better state than before.
+
+There are three aspects to reviewing we will focus on:  
+
+1. Identify areas in the code and documentation that are opportunities for improvement.
+2. Communicate your questions and concerns effectively and in a way that creates a positive atmosphere.
+3. Determine solutions collaboratively in a way that allows for a learning as well as a long term improved product.
+
+### What to look for!
+
+Depending on the goals of the project, and pull request there can be a lot to keep an eye out for. There are [many articles out there about what to look for in a code review](https://github.com/joho/awesome-code-review#articles).
+
+_Here's some general points:_  
+
+- Does the analysis answer the question it's asking? Are the methods it uses to do so appropriate?
+- Is the code clear and readable? Does it contain a healthy amount of comments and documentation for individuals not familiar with the project to understand generally what is going on?
+- Is the code efficient with computational resources? (Are there areas it's a bit too greedy with memory usage?)
+- Does the code stick to the style and conventions of this project?
+- Are there alternate scenarios where the current strategy might fail? (depending on the likelihood of this use case, this may be an instance for a new issue and for it to be addressed in a different pull request).
```

Comment:
```suggestion
- Are there alternate scenarios where the current strategy might fail? (depending on the likelihood, this may be an instance for a new issue and another pull request).
```

Reply:
simplifying for readability